boards/arm/ti_am62x_sk_m4/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# Texas Instruments Sitara AM62x-SK-M4 EVM
+#
+# Copyright (c) 2023 L Lakshmanan
+#
```

Comment:
Here and other places, use a TI copyright statement as this is  part of an official TI project.
`Copyright (C) 2023 Texas Instruments Incorporated`

Reply:
@vaishnavachath isn't that only the case if the contributor is employed by TI?

Reply:
> @vaishnavachath isn't that only the case if the contributor is employed by TI?

In this case, Lakshmanan (the contributor) is interning at TI and you can see all his commits are signed off by @ti.com domain.

Hence  I believe Vaishnav has made the above suggestion.

Reply:
Fixed in all files now.